### ISSUE TITLE
Why is the public interface of "MPFullscreenAdAdapter.hasAdAvailable" writable?

### ISSUE THREAD

**Issue by im-ttmskk** [mopub/mopub-ios-sdk#326]
I don't know if the adapter needs to accept availability changes externally.

- [x] I am submitting a bug report for existing functionality
- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### MoPub SDK Version:

5.13.0

#### Device model and OS Version:

Unrelated

#### Xcode Version:

Xcode 11

#### Ad Unit IDs used in reproducing the issue:

Unrelated

#### Steps to reproduce the behavior:

https://github.com/mopub/mopub-ios-sdk/blob/e285dceb7bdc4773966e0e6a8f780a4690cdb0aa/MoPubSDK/MPFullscreenAdAdapter.h#L61

#### Expected behavior:

Write:
```objc
- (BOOL) hasAdAvailable {
    return self.thirdPartyAd.isReady
}
```

#### Observed behavior:

Write:
```objc
- (BOOL) hasAdAvailable {
    return self.thirdPartyAd.isReady
}
- (void) setHasAdAvailable:(BOOL)hasAdAvailable {
    // NOOP
}
```

#### Evidence:
Device log files, Network log file, etc.

**Comment by chauduyphanvu** [comment]
@im-ttmskk, thanks for letting us know! I've relayed this to the rest of the team, and we will be addressing this in an upcoming SDK version. Let us know if you have any questions!

Vu Chau
MoPub Team

**Comment by chauduyphanvu** [comment]
This has been fixed in [v5.14.0](https://github.com/mopub/mopub-ios-sdk/commit/5bcd84e86ab5d4c41e802d39bdc4a7cc84d97433). Update and let us know if you continue seeing the issue!
